actions/AuthNAction.js

```diff
@@ -64,12 +64,15 @@ async function getAccessToken(params) {
   ) {
     const accessToken = params.__ow_headers.authorization.substring('Bearer '.length);
     // Validate the access token
+
+    // The error identifiers (enclosed in double curly braces) will be replaced with the actual
+    // error messages after being processed for localization on the frontend
     if (!await isValidToken(imsEndpoint, clientId, accessToken)) {
-      throw new Error('The access token is not valid');
+      throw new Error('{{invalidAccessToken}}');
```

Comment:
In general, API responses should be self-explanatory, at least for the developers who use them, and optionally provide some additional data to more closely identify error responses. This could include a link to documentation or an internal error code that might be mapped to the error. Clients (our web app in this case) might use this information to properly localize and format a message shown to end users.

I believe we agreed to use this [approach](https://github.com/adobe/aem-genai-assistant/pull/282#discussion_r1621457969). In this PR, as I understand, the returned error message will be:
```
{
  "error": "{{invalidAccessToken}}"
}
```
?

Reply:
Yes, I suggested this method as the original error message needs to live in the frontend code for the localization to work correctly.